Source/monster.cpp

```diff
@@ -2405,7 +2405,7 @@ void SkeletonBowAi(int i)
 
 	if (abs(mx) < 4 && abs(my) < 4) {
 		if ((monster._mVar2 > 20 && v < 2 * monster._mint + 13)
-		    || ((monster._mVar1 == MM_WALK || monster._mVar1 == MM_WALK2 || monster._mVar1 == MM_WALK3)
+		    || (IsAnyOf(static_cast<MonsterMode>(monster._mVar1), MonsterMode::MoveNorthwards, MonsterMode::MoveSouthwards, MonsterMode::MoveSideways)

```

Comment:
Having an `IsWalking()` method like on player would help to clean up this pattern.

Reply:
There is already one, but it deals with the actual state of the monster. `_mVar1` and `_mVar2` are not the current state. This is the same as what @ephphatha commented previously and I already replied back.

Reply:
I thought that was more about how to handle `var1`, I actually didn't really notice that it was var1 and not `MMode` when making this comment.